modules/EnsEMBL/Web/File/Utils/TrackHub.pm

```diff
@@ -157,8 +157,8 @@ sub get_hub {
           $response = read_file($file, $file_args); 
 
           if ($response->{'error'} || !$response->{'content'}) {
-            my $error = @{$response->{'error'}} || "trackDB file empty for genome $genome";
-            push @errors, "$genome ($file): $error";
+            my $error = $response->{'error'}[0] || "trackDB file empty for genome $genome";
+            push @errors, $error;
```

Comment:
Just to be sure - I see from changes in URL.pm that for remote files the URL will be included in $error, however will the messages for local ones be detailed enough following removal of $genome and $file here?

Reply:
Yes - turns out the error message already contains that information, so including it again made it less readable. This is a message displayed in the browser to help the user diagnose problems with file uploads, so it needs to be as simple as possible.

Reply:
Also, that second change (line 161) is not a message from a local file but from a remote trackhub. The trackhub parser provides fairly comprehensive error messages to aid in debugging.